dev/bots/analyze.dart

```diff
@@ -2460,6 +2463,66 @@ Future<void> verifyTabooDocumentation(String workingDirectory, {int minimumMatch
   }
 }
 
+final Map<String, String> _kKotlinTemplateKeys = {
+  'androidIdentifier': 'dummyPackage',
+  'pluginClass': 'PluginClass',
+  'projectName': 'dummy',
+  'agpVersion': '0.0.0.1',
+  'kotlinVersion': '0.0.0.1',
+};
+
+final String _kKotlinTemplateRelativePath = path.join(
+  'packages',
+  'flutter_tools',
+  'templates',
+);
+
+const List<String> _kKotlinExtList = <String>['.kt.tmpl', '.kts.tmpl'];
+const String _kKotlinTmplExt = '.tmpl';
+final RegExp _kKotlinTemplatePattern = RegExp(r'{{(.*?)}}');
+
+Future<void> lintKotlinTemplatedFiles(String workingDirectory) async {
```

Comment:
Can you add a dart doc to this method that gives an easy to read overview. 

Maybe something like 
```
Copy kotlin template files from [_kKotlinTemplateRelativePath] into [workingDirectory] then replace template values with values from [_kKotlinTemplateKeys] or "'dummy'" if an unknown key is found. 
```

Reply:
Done :)